FreeRTOS/Demo/WIN32-64-MSVC/main.c

```diff
@@ -80,9 +80,9 @@
  * as this demo could easily create one large heap region instead of multiple
  * smaller heap regions - in which case heap_4.c would be the more appropriate
  * choice.  See http://www.freertos.org/a00111.html for an explanation. */
-#define mainREGION_1_SIZE                     8201
-#define mainREGION_2_SIZE                     23905
-#define mainREGION_3_SIZE                     16807
+#define mainREGION_1_SIZE                     82010
```

Comment:
Why is this change needed?

Reply:
When running on x64 there wasn't enough heap. I've simply x10 all of those.